Resources/Computing/ARCComputingElement.py

```diff
@@ -375,11 +375,19 @@ def getJobStatus(self, jobIDList):
         job = j
       jobList.append(job)
 
-    resultDict = {}
+    jobs = []
     for jobID in jobList:
+      jobs.append(self.__getARCJob(jobID))
+
+    # JobSupervisor is able to aggregate jobs to perform bulk operations and thus minimizes the communication overhead
+    job_supervisor = arc.JobSupervisor(self.usercfg, jobs)
+    job_supervisor.Update()
```

Comment:
2 questions:

- do we need a certain version of `arc` for having this? What we have in DIRACOS is enough?
- what are the other functions of `JobSupervisor`? E.g. does it have a `cancel` method (that we could use for killing the jobs)?

Reply:
* I haven't found any specification on this point in the documentation, just a Git repository containing the class in 2012 and I tested it with the version available in DIRACOS.
* Yes it has a `cancel` method that could be useful. Should we use this one? It also has a `retrieve` and a `renew` method but I don't think it will help in our case.

Reply:
I understand that the JobSupervisor is supported in essentially all versions of arc. So, the stuff in DIRACOS should be enough. More information on this at

http://www.nordugrid.org/documents/code/sdk/classArc_1_1JobSupervisor.html

Reply:
I think that if you use `cancel` the code could be simplified further, so I'd go for it.

Reply:
Alright I agree, I have done the change. 
We just have lost accuracy to debug the case where a specific job cannot be cancelled but I guess it is not a big issue.

Reply:
Alright, I agree, I have done the change.
We just have lost accuracy to debug the case where a specific job cannot be cancelled but I guess it is not a big deal.

Reply:
Good. BTW, what the `retrieve` and `renew` methods do?

Reply:
* `retrieve` is used to get the output of a given job or group of jobs. In our context, it is used in `getJobOutput` where we only have one jobID in parameter so this is not useful to replace `Job` by `JobSupervisor` in this case.
* `renew` is used to renew the credentials of a job or group of jobs. In out context, we only use the method when it is necessary, I've just replaced the `Job.renew` by a `JobSupervisor.renew`. I aggregate the jobs to renew and the jobs to cancel in a list and at the end I perform a bulk operation using the `JobSupervisor` (see in `getJobStatus`).